content/it/docs/concepts/containers/container-environment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Container Environment
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Questa pagina descrive le risorse disponibili nei Containers eseguiti in Kubernetes.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Container environment
+
+Quando si esegue un Container in Kubernetes, le seguenti risorse sono rese disponibili:
+
+* Un filesystem, composto dal file system dell'[image](/docs/concepts/containers/images/) e da uno o più [volumes](/docs/concepts/storage/volumes/).
+* Una serie di informazioni sul container stesso.
+* Una serie di informazioni sugli oggetti nel cluster.
+
+### Informazioni sul Container
+
+L' *hostname* di un container è il nome del Pod all'interno del quale è eseguito il container.
+E' consultabile tramite il comando `hostname` o tramite la funzione
+[`gethostname`](http://man7.org/linux/man-pages/man2/gethostname.2.html)
+disponibile in libc.
+
+Il Pod name e il namespace possono essere resi disponibili come environment variables attraverso l'uso
```

Comment:
s/Pod name/nome del Pod/
Is environment variables a technicla term or should it be trnaslated to "variabili d'ambiente"?

Reply:
I will consider it a technical term